components/actionbutton/skin.css

```diff
@@ -0,0 +1,340 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+:root {
+  /* Overridden because of the way we draw focus rings */
+  --spectrum-actionbutton-m-quiet-border-size-key-focus: 1px;
+}
+
+.spectrum-ActionButton {
+  background-color: var(--spectrum-actionbutton-m-background-color);
+  border-color: var(--spectrum-actionbutton-m-border-color);
+  color: var(--spectrum-actionbutton-m-text-color);
+
+  .spectrum-Icon {
+    color: var(--spectrum-actionbutton-m-icon-color);
+  }
+
+  .spectrum-ActionButton-hold {
+    color: var(--spectrum-actionbutton-m-hold-icon-color);
+  }
+
+  &:hover {
+    background-color: var(--spectrum-actionbutton-m-background-color-hover);
+    border-color: var(--spectrum-actionbutton-m-border-color-hover);
+    color: var(--spectrum-actionbutton-m-text-color-hover);
+
+    .spectrum-Icon {
+      color: var(--spectrum-actionbutton-m-icon-color-hover);
```

Comment:
Just curious what is the meaning of `-m-` here? Is it `medium`, why even color has a medium size?

Reply:
Yeah, it's medium. All variables are re-defined for each t-shirt size, and `--spectrum-actionbutton-icon-color-hover` is no longer defined. We have to use `--spectrum-actionbutton-m-icon-color-hover`.